pkg/cvo/status.go

```diff
@@ -159,6 +159,9 @@ const ClusterVersionInvalid configv1.ClusterStatusConditionType = "Invalid"
 // and no failures occurred during image verification and precondition checking.
 const DesiredReleaseAccepted configv1.ClusterStatusConditionType = "ReleaseAccepted"
 
+// ImplicitlyEnabledCapabilities
```

Comment:
```go
// ImplicitlyEnabledCapabilities is True if there are enabled capabilities which the user is not currently
// requesting via spec.capabilities, because the cluster version operator does not support uninstalling
// capabilities if any associated resources were previously managed by the CVO.
```

or some such?